dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -96,9 +96,20 @@
       <AdditionalDependencies>$(OutDir)..\mrmmin\mrmmin.lib;rpcrt4.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <ModuleDefinitionFile>MRM.def</ModuleDefinitionFile>
       <SubSystem>Windows</SubSystem>
+      <!-- This sets /DEBUG -->
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <!-- Note: /PROFILE implies the following 4 settings.
+           However, since /PROFILE is only supported on VS Enterprise we will also set them explicitly below as well. -->
```

Comment:
Would it be more helpful to define something in a root props and use it where needed, e.g. create `/Profile.props` that does lines 101-112 and then in this file just `<Import...Profile.props/>` ?

Reply:
Good idea!
What about something like `WindowsAppSDK.Vulcanize.Cpp.props` in the top-level dev folder (since this is specific to C++) ?

Reply:
Per offline conversation with Rohan, going to use `WindowsAppSDK.ApiScan.Cpp.props` instead.

Reply:
If Vulcanize is C/C++ specific then yes on the .Cpp part

I'd omit the WindowsAppSDK prefix. We have other *.props in the  root and non have the project name as a prefix. So first pass I suggest `\Vulcanize.Cpp.props`, and think about <del>if</del> when we refactor what should the structure look like? Should the be a subdir? Called what? `/build` is for the ADO build pipeline. Should there be a subdir under and existing directory or a new root? `/build/msbuild` or...?

So go with Vulcanize.Cpp.props for now to unblock this work, and then cleaning up file structure is a simple (later) PR.

Reply:
Thanks!
Yes, the Vulcanization settings are specific to C/C++. There's different ones for C#/Managed code, so we'd likely want to have that be a separate file. Or we could probably share the same file, but since this only handles C/C++ for now, it feels like we'd want to denote that with adding `.Cpp` to the name.

I'll move the file to the root level folder and use the name `ApiScan.Cpp.props` (so no prefix of WindowsAppSDK). Also using ApiScan instead of Vulcanize, in order to avoid using internal-only names.